CS/VehicleDekho/src/com/vehicledekho/core/helper/VehicleHelper.java

```diff
@@ -0,0 +1,46 @@
+package com.vehicledekho.core.helper;
+
+import com.vehicledekho.core.Bike;
+import com.vehicledekho.core.Car;
+import com.vehicledekho.core.Vehicle;
+
+public class VehicleHelper {
+	public static Vehicle createVehicle(String vehicleType, String model,
+			String make) {
+		if (vehicleType.equalsIgnoreCase("car")) {
+			return createCar(model, make);
+		} else {
+			return carBike(model, make);
+		}
+	}
+
+	private static Vehicle carBike(String model, String make) {
+		Bike bike = new Bike();
+		bike.setModel(model);
+		bike.setMake(make);
+		bike.setSelfStart(true);
```

Comment:
it was just an prototype to make object...even if they are similar that wont affect anything...